### ISSUE TITLE
The example of esp-dl/tools/tensorflow_to_onnx does not support MatMul and Softmax yet (AIV-536)

### ISSUE THREAD

**Issue by QimingLee** [espressif/esp-dl#98]
We complete the  example and run the program, but it shows "Generating the quantization table:
MatMul is not supported on esp-dl yet
Softmax is not supported on esp-dl yet"

Traceback (most recent call last):
  File "D:/esp-dl-master/esp-dl-master/tools/quantization_tool/examples/example.py", line 39, in <module>
    calib.generate_quantization_table(model_proto, calib_dataset, pickle_file_path)
  File "calibrator.py", line 333, in calibrator.Calibrator.generate_quantization_table
  File "calibrator.py", line 222, in calibrator.Calibrator.generate_output_model
  File "calibrator.py", line 217, in calibrator.Calibrator.check_model
ValueError: current model is not supported by esp-dl

**Comment by Raven888888** [comment]
@Auroragan when can it support more layers like MatMul and Clip in calibrator? These are very common layers and the calibrator is too outdated

**Comment by Farzinkh** [comment]
Hi, I was successful in implementing Convolution,MatMul and Softmax [check out here](https://github.com/Farzinkh/AI_EDGE/tree/main/ESP-Dl/Hand_gesture_recognition).

**Comment by Raven888888** [comment]
@Farzinkh impressive! I sure will try out!

**Comment by SamsonAdem** [comment]
Is this resolved??

**Comment by AlanG2004** [comment]
> Hi, I was successful in implementing Convolution,MatMul and Softmax [check out here](https://github.com/Farzinkh/AI_EDGE/tree/main/ESP-Dl/Hand_gesture_recognition).

Where specifically should I look? The repository you provided is hard to follow, please let me know I am having the same issue and it is frustrating. Thank you.

**Comment by AlanG2004** [comment]
> We complete the example and run the program, but it shows "Generating the quantization table: MatMul is not supported on esp-dl yet Softmax is not supported on esp-dl yet"
> 
> Traceback (most recent call last): File "D:/esp-dl-master/esp-dl-master/tools/quantization_tool/examples/example.py", line 39, in calib.generate_quantization_table(model_proto, calib_dataset, pickle_file_path) File "calibrator.py", line 333, in calibrator.Calibrator.generate_quantization_table File "calibrator.py", line 222, in calibrator.Calibrator.generate_output_model File "calibrator.py", line 217, in calibrator.Calibrator.check_model ValueError: current model is not supported by esp-dl

Hello were you able to solve this problem? I am running into the same issue any help would be appreciated @QimingLee @Farzinkh

**Comment by sun-xiangyu** [comment]
Please try new esp-dl. Now MatMul and Softmax have been supported.
